src/app/beer_garden/garden.py

```diff
@@ -130,7 +130,7 @@ def handle_event(event):
     This method should NOT update the routing module. Let its handler worry about that!
     """
     if event.garden != config.get("garden.name"):
-        if event.name in (Events.GARDEN_STARTED.name, Events.GARDEN_UPDATED.name,):
+        if event.name in (Events.GARDEN_STARTED.name, Events.GARDEN_UPDATED.name):
```

Comment:
I think this is causing a merge conflict...

I'm not *opposed* to changing stuff like this (especially if it fixes the linting) but I think for things like this we should just make the change directly in `v3` and push it up. That way we don't complicate PRs with "extra" changes. I think we can skip the review for this stuff :smile:

Reply:
I agree, I normally try to not check in these formatting  changes when it doesn't impact my code. This one got slipped in on accident. I will revert this one.